packages/react-native-client/android/src/main/java/io/fishjam/reactnative/VideoPreviewViewModule.kt

```diff
@@ -9,6 +9,11 @@ class VideoPreviewViewModule : Module() {
       Name("VideoPreviewViewModule")
 
       View(VideoPreviewView::class) {
+        OnViewDestroys { view: VideoPreviewView ->
+          view.dispose()
+          RNFishjamClient.localTracksSwitchListenerManager.remove(view)
```

Comment:
**question:** Why is this needed? `onDetachedFromWindow` is not enough?

Reply:
I wil explain irl.